compiler/test_evaluation_scripts.sh

```diff
@@ -5,16 +5,21 @@ export PASH_TOP=${PASH_TOP:-$(git rev-parse --show-toplevel --show-superproject-
 export DEBUG=0
 # export DEBUG=1 # Uncomment to print pash output
 
-microbenchmarks_dir="${PASH_TOP}/evaluation/microbenchmarks/"
-intermediary_dir="${PASH_TOP}/evaluation/test_intermediary/"
-test_results_dir="${PASH_TOP}/evaluation/results/test_results/"
+microbenchmarks_dir="${PASH_TOP}/evaluation/tests/"
+intermediary_dir="${PASH_TOP}/evaluation/tests/test_intermediary/"
+test_results_dir="${PASH_TOP}/evaluation/tests/results/"
 
 echo "Deleting eager intermediate files..."
 rm -f /tmp/eager*
 mkdir -p $intermediary_dir
 rm -rf "$test_results_dir"
 mkdir -p "$test_results_dir"
 
+echo "Generating inputs..."
+cd "$microbenchmarks_dir/input"
+./setup.sh
```

Comment:
@nvasilakis so I added this here to make sure that inputs are generated before running tests, but we wouldn't want to re-download inputs whenever we run tests. How should we improve that? Should we check if the files exist before running the setup (maybe having a check-setup) file or should we just run the setup for tests in the installation script?

Reply:
That's a great point, and we do some of that in other `setup.sh` scripts. Probably the best way to handle this would be in `setup.sh` script itself (cc @dkarnikis). This way every `setup.sh` would be self-contained, and we don't have to remember to call more scripts or check values. I just updated `setup.sh` to handle this. What do you think?